unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -953,7 +953,19 @@ public class SaveAvatarPayload
             public bool isSignUpFlow;
             public AvatarModel avatar;
         }
-
+        
+        public static class RendererAuthenticationType
+        {
+            public static string Guest => "guest";
+            public static string WalletConnect => "WalletConnect";
```

Comment:
this string is not matching what is expected on kernel side, which is `'wallet_connect'`
other than that the looks great

Reply:
ups :fearful: